### PR TITLE
Use a different React app starter + cleanup + small fixes

### PR DESCRIPTION
**Pull request by alansemenov**

### CODE REVIEW

File: docs/react-app.adoc [hunk 1]
```diff
@@ -6,62 +6,60 @@ In this chapter you will create a React application with TypeScript support.
...
-REACT_APP_GUILLOTINE_URL=<Query Playground URL>/master
+VITE_GUILLOTINE_URL=http://localhost:8080/site/intro/master
 ----
-Replace `<Query Playground URL>` with the URL you used to reach Query Playground when setting up Enonic.
+which is the URL you used to reach Query Playground when setting up Enonic, with appended `/master`.
 
 NOTE: `/master` is added to the URL because we want to access the master branch (containing the published content)
 
-Wonder why it is called Guillotine? Guillotine is simply the name for Enonic's headless GraphQL API.
+Wonder why it is called Guillotine? Guillotine is simply the name for Enonic's headless https://developer.enonic.com/docs/guillotine/stable[GraphQL API].
```
Comment (sigdestad) [inline docs/react-app.adoc:45]: We should normally not include the /stable extension when making links to docs

File: docs/media/react-app.png
Comment (sigdestad) [inline docs/media/react-app.png:1]: Can you grab a smaller screenshot? This will not look good on mobile

File: docs/react-app.adoc
Comment (sigdestad) [inline docs/react-app.adoc:1]: Why would we suddenly change this from React to "vite?"?

**Review by sigdestad — APPROVED** [review]
Se gjerne på kommentarene mine

File: docs/react-app.adoc
Comment (alansemenov) [reply docs/react-app.adoc:1]: It's still React. We just use Vite starter to create a new React app. The one we originally used - [react-scripts](https://www.npmjs.com/package/react-scripts) - hasn't been released for 2 years and, as a result of bad maintenance, is giving warnings about high and medium vulnerabilities when creating an app.

**Review by pmi — APPROVED** [review]

**Review by ComLock — APPROVED** [review]

**Pull request merged**